packages/playground/src/App.vue

```diff
@@ -235,6 +274,13 @@ function clickHandler({ route, url }: AppRouteItem): void {
 }
 
 $router.beforeEach((to, from, next) => {
+  if (network !== "main") {
+    routes = routes.filter(route => route.title !== "Minting");
+    if (to.path === "/minting") {
+      return;
+    }
+    next();
+  }
   if (to.path === "/" && hasActiveProfile) {
     next({ path: "portal/twin" });
```

Comment:
Please don't update routes in runtime .... define a guard for minting guard to redirect or whatever you need if network is main